docs/advanced/scf/hsolver.md

```diff
@@ -4,7 +4,7 @@
 
 Method of explicit solving KS-equation can be chosen by variable "ks_solver" in INPUT file.
 
-When "basis_type = pw", `ks_solver` can be `cg` or `dav`. The default setting `cg` is recommended, which is band-by-band conjugate gradient diagonalization method. There is a large probability that the use of setting of `dav` , which is block Davidson diagonalization method, can be tried to improve performance.  
+When "basis_type = pw", `ks_solver` can be `cg`, `bpcg` or `dav`. The `bpcg` method only supports K-point parallelism currently. The default setting `cg` is recommended, which is band-by-band conjugate gradient diagonalization method. There is a large probability that the use of setting of `dav` , which is block Davidson diagonalization method, can be tried to improve performance.  
```

Comment:
I am interesting that whether the suggestion to improve performance should update with bpcg implementation.

Reply:
I suggest keeping the current recommendations. Because BPCG is new, it needs more testing to make it more stable and user-friendly. We also need more tests to confirm its performance improvement.

Reply:
Thank you for your reply.